Remotion/Web/Development.WebTesting/HostingStrategies/Configuration/TestSiteLayoutConfiguration.cs

```diff
@@ -33,13 +33,13 @@ public class TestSiteLayoutConfiguration : ITestSiteLayoutConfiguration
     /// <inheritdoc />
     public IReadOnlyList<ITestSiteResource> Resources { get; }
 
-    public TestSiteLayoutConfiguration ([NotNull] WebTestConfigurationSection configSettings)
+    public TestSiteLayoutConfiguration ([NotNull] IWebTestSettings configSettings)
```

Comment:
`configSettings` no longer hits the mark. I think `webTestSettings` would be best as it fits with the naming in the other classes.